core/vm/evm.go

```diff
@@ -63,22 +63,23 @@ func (evm *EVM) precompile(addr common.Address) (CeloPrecompiledContract, bool)
 		precompiles = PrecompiledContractsHomestead
 	}
 	p, ok := precompiles[addr]
+	if evm.Config.PrecompileOverrides != nil {
```

Comment:
If this is non-nil now, the Celo precompiles will be skipped.

IOW, I think the case where `evm.Config.PrecompileOverrides != nil` and `addr = <TRANSFER_PRECOMPILE>` would not return the transfer precompile as expected.

Reply:
Ah yes, I did the same error as before just in the opposite direction. I guess my patience was running low after finally locating the problem. (This is was made `./op-chain-ops/script TestScript` fail)

I changed the override part to set `p` and `ok` rather than returning, so that the celo precompiles will be considered if no override was found.